Create 08-09-2021_writeup.md (#99)
very-good-science/data-ethics-club: site/contents/write_ups/2021/08-09-21_writeup.md

There is an intersection now for AI questions for ethical questions and protecting participants and their data. 

Agreeing on these sort of things unanimously is impossible, but people trying to remove personal bias is important, i.e. not doing ethical reviews on projects with military funding if you're unable to be unbiased there. Perhaps we need to start sooner with considering ethical frameworks as early as undergrad research, or at least in PhDs.

How many hoops do you need to jump through, two separate ethical reviews to go through simultaneously until you satisfy both sets of requirements.

The paper doesn't evaluate the process in terms of intended outcome, just on how the participants felt. So perhaps that means we need another review system.

Perhaps the ethical reviews need to be enforced across funding groups, otherwise researchers might pick and choose funding boards so they have the easiest time hopping through hoops.

## On the Consequences and Moral Character of Science

> Consequences of science are very complex and difficult to predict. There is a problem then, with the idea that we can judge science against societal goals. We have very little consesus on this.

Reviewer: ```suggestion
Consequences of science are very complex and difficult to predict. There is a problem then, with the idea that we can judge science against societal goals. We have very little consensus on this.
```

> One piece of work that touches on this is ["The Moral Character of Crytopgrahic Work"](https://web.cs.ucdavis.edu/~rogaway/papers/moral-fn.pdf) by Philip Rogaway. In it, Rogaway that "cryotopgrahy rearranges power" and so as a result is not only "an inherently political tool", but it confers "an intrisically moral dimension" to the field of crytopgrahy. He later goes onto argues that the "inability to effectively address mass surveillance constitutes a failiure of our field". He finishes the abstract with the following sentence: "I plead for a reinvention of our disciplinary culture to attend not only to puzzles and math, but, also, to the societal implications of our work.""

Reviewer: ```suggestion
One piece of work that touches on this is ["The Moral Character of Cryptographic Work"](https://web.cs.ucdavis.edu/~rogaway/papers/moral-fn.pdf) by Philip Rogaway. In it, Rogaway states that "cryptography rearranges power" and so as a result is not only "an inherently political tool", but it confers "an intrinsically moral dimension" to the field of cryptography. He later goes onto argues that the "inability to effectively address mass surveillance constitutes a failure of our field". He finishes the abstract with the following sentence: "I plead for a reinvention of our disciplinary culture to attend not only to puzzles and math, but, also, to the societal implications of our work.""
```
Just a couple of typos :)

We should not criminalise behaviours that lead to crimes, we should only criminalise the crimes themselves. Being in poverty and starving might lead to stealing food, but we should not criminalise being in poverty. We should not use laws to control behaviours. Some in the group argued we should approach ethical processes in the same way.

We can't ask of every scientific advance, "what are the implications?" That's too complicated. Some in the group argued, instead we should concentrate on making sure society is armed with the tools to control these bad things which will inevitably happening. We can't control a tech firm using our AI research to be racist. We should not blame ourselves for being curious, we should blame the tech firm for being bad. 

Maybe it's hard to separate the end use with the intention. There exists dual use technology, such as a stress recognition system that may not have solely good intentions. Perhaps you should design it with consideration.

> Not to calculate the moral worth of one decision over another, but at least considering intuitive moral decision making around some principles. But what principles? Can we ever agree on them? Academics need to be more humble and a bit more measured in what they say they can do.

Reviewer: ```suggestion
Not to calculate the moral worth of one decision over another, but at least considering intuitive moral decision making around some principles. But what principles? Can we ever agree on them? Academics need to be more humble and a bit more measured in what they say they can do (Ismael Kherroubi Garcia has done [some excellent writing on this idea](https://towardsdatascience.com/data-science-meaning-and-diversity-bb842602e55d) in data science, framed as 'epistemic humility')
```

> What counts as positive impact? Add ESR into the IRB rather than as a separate thing? Then there are other separate reforms that

Reviewer: ```suggestion

```
Maybe remove the whole sentence? Not sure what the end point would have been
